docs/s3-bucket-versioning.md

```diff
@@ -0,0 +1,78 @@
+# **Steps to Create and Configure a Versioned S3 Bucket**
+
+## **Set a Variable for the Bucket Name**
+```bash
+BUCKET_NAME=temp-bucket-for-versioning
+```
+
+## **Create a New S3 Bucket**
+Use the following command to create a bucket. Make sure to specify the correct region.
+
+```bash
+aws s3api create-bucket --bucket $BUCKET_NAME --region your-region
+```
+
+---
+
+## **Enable Versioning on the Bucket**
```

Comment:
How can I see/track the changes/versions?
When should I change the version?
Would be good to add a section about it too.

Reply:
To track changes/versions, I have added a brief section down in **Verify Configuration**
Version changes happen automatically with each update or delete marker, so manual version changes aren't needed.